bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/discovery/MieleMDNSDiscoveryParticipant.java

```diff
@@ -70,83 +79,88 @@ public void modified(@Nullable Map<String, Object> configProperties) {
      * @param configProperties the passed configuration parameters.
      */
     private void updateRemovalGracePeriod(Map<String, Object> configProperties) {
```

Comment:
You are now calling this method with a nullable map so you should mark the param as @nullable and keep your test.

Reply:
I don't see a point in calling a method with null just so it can do nothing? `activate` and `modified` are declared with `@Nullable` so can't do anything about that, but I can at least avoid propagating that further.

Reply:
Oh wait, now read your comment again. I think you didn't see the changes in `activate` and `modifed`? `updateRemovalGracePeriod` is no longer called with a nullable map. This would result in warnings, as its not declared nullable anymore.

Reply:
activate and modified could be called with a null parameter.

`public void activate(@Nullable Map<String, Object> configProperties) {`

Reply:
Ok, I thought the null test in `activate` and `modified` was removed, while in fact it is added.
So of course, it is ok.